Subject: Avoid "warning: target CPU does not support interworking"
From: Juergen Beisert <j.beisert@pengutronix.de>

This patch is required only when using an iwmmxt optimised toolchain.
U-Boot does not support EABI, so some options must be modified to
avoid confusing messages.

Intended for mainline? Not sure. But it should be used for u-boot-v2!

Signed-off-by: Juergen Beisert <j.beisert@pengutronix.de>

---
 Makefile |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

Index: cpu/pxa/config.mk
===================================================================
--- cpu/pxa/config.mk.orig
+++ cpu/pxa/config.mk
@@ -25,8 +25,7 @@
 PLATFORM_RELFLAGS += -fno-strict-aliasing  -fno-common -ffixed-r8 \
 	-msoft-float
 
-#PLATFORM_CPPFLAGS += -mapcs-32 -march=armv4 -mtune=strongarm1100
-PLATFORM_CPPFLAGS += -march=armv5 -mtune=xscale
+PLATFORM_CPPFLAGS += -march=armv5t -mtune=xscale
 # =========================================================================
 #
 # Supply options according to compiler version
